packages/framer-motion/src/utils/mix-color.ts

```diff
@@ -26,10 +29,10 @@ function asRGBA(color: Color | string) {
     let model = type!.parse(color)
 
     if (type === hsla) {
-        model = hslaToRgba(model)
+        model = hslaToRgba(model as HSLA)
```

Comment:
Why is this cast now needed?

Reply:
There's some extra casts in there because I believe Framer Motion is compiled with stricter settings than Popmotion was

Reply:
Makes sense, can we add a todo to try to remove them?